cmd/local-gadget/containers/containers.go

```diff
@@ -92,30 +91,18 @@ func NewListContainersCmd() *cobra.Command {
 
 			parser.SortContainers(containers)
 
-			switch commonFlags.OutputMode {
-			case commonutils.OutputModeJSON:
-				b, err := json.MarshalIndent(containers, "", "  ")
-				if err != nil {
-					return commonutils.WrapInErrMarshalOutput(err)
-				}
```

Comment:
I have the same concern as before. Why not to keep this behavior and print a json array with all the containers??

Reply:
🤦🏽 ... Yep, I had to revert these changes as well. I am sorry!